README.md

```diff
@@ -98,7 +98,7 @@ vagrant up
 
 Then, run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To release a new version, update the **version number** in `logstash-filter-fix_message_filter.gemspec`, and then run `bundle exec rake release` to create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To release a new version, update the **version number** in `logstash-filter-protocol.gemspec`, and then run `bundle exec rake release` to create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
```

Comment:
@cbusbey - I can't make changes directly to your PR, otherwise I would for this, but this one needs-a-fixin